src/front_input.c

```diff
@@ -1289,6 +1289,27 @@ short get_creature_control_action_inputs(void)
             }
             message_add(CrInst, get_string(StrID));
         }
+        if (is_key_pressed(KC_SEMICOLON,KMod_NONE))
+        {
+            char CrInst = -70;
+            if (creature_instance_is_available(creatng, CrInst_FIRST_PERSON_DIG))
+            {
+                if (game.active_messages_count > 0)
+                {
+                    clear_messages_from_player(CrInst);
+                }
+            }
+            first_person_dig_claim_mode ^= 1;
+            if (first_person_dig_claim_mode)
+            {
+                message_add_fmt(CrInst, "Claim mode");
```

Comment:
Adding English text is not the way to go. Just a minority of the people play this game in English.